### ISSUE TITLE
[SUGGESTION] Strict Jesus

### ISSUE THREAD

**Issue by 453452hw** [MeteorDevelopment/meteor-client#556]
**Describe the feature**
idk some sort of jesus that works on ecme. cause aperently you need "strict jesus for that"

**Screenshots**
no

**Additional info**
hello person reading this i love you no homo

**Comment by supakeks** [comment]
ncp jesus

**Comment by RedCarlos26** [comment]
Yea

**Comment by Cypphi** [comment]
Dolphin would also be nice for the dolphin fags :trollface:

Ignore that but I actually agree, jesus should have more options and stuff because we're currently stuck to a basic jesus, like it works, but most anticheats go :sob: when using it

**Comment by Lerbiq** [comment]
Yeah, updated NCP (1.12.2+) just rubberbands if you try to walk on water. Even with waterwalk permissions.

**Comment by supakeks** [comment]
Yeah is there even any way to use jesus on updated ncp even with the waterwalk permissions? (other than using strict jesus)

**Comment by Lerbiq** [comment]
Yes. [JaxClient's jesus](https://github.com/DustinRepo/JexClient-main/blob/main/src/main/java/me/dustin/jex/module/impl/player/Jesus.java) on solid mode works on updated NCP (at least with my server's NCP config). I'm gonna test with default config later.

**Comment by Lerbiq** [comment]
Alright so it works even with default config. However if the player doesn't have `nocheatplus.checks.moving.survivalfly.waterwalk` permission NCP will rubberband them.

**Comment by Declipsonator** [comment]
what if jesus slightly dipped into the water
